blivet/devices/stratis.py

```diff
@@ -48,6 +48,10 @@ def _destroy(self):
         log_method_call(self, self.name, status=self.status)
         devicelibs.stratis.remove_pool(self.uuid)
 
+    def dracut_setup_args(self):
+        return set(["stratis.rootfs.pool_uuid=%s" % self.uuid] +
+                   ["stratis.rootfs.uuids_paths=/dev/disk/by-uuid/%s" % p.uuid for p in self.parents])
```

Comment:
One note here is that the dracut command line argument requires a UUID format with dashes (this is how the symlinks are generated by udev) but our D-Bus API exports UUIDs without dashes, only hex characters. We already have an issue out about this: 

stratis-storage/stratis-cli#698

For now, you may want to parse the UUID format returned from the D-Bus and ensure that the output is returned with dashes if that's possible.

cc @mulkieran